packages/ui-toggle-details/src/ToggleDetails/index.tsx

```diff
@@ -152,11 +155,11 @@ class ToggleDetails extends Component<ToggleDetailsProps> {
   }
 
   renderDetails(expanded: boolean, detailsProps: { id: string }) {
-    const { children } = this.props
+    if (!this.props.children) return null
     const expandedStyles = expanded ? { display: 'block' } : { display: 'none' }
     return (
       <div {...detailsProps} css={[this.props.styles?.details, expandedStyles]}>
```

Comment:
This `div` is rendering a line with 0.375rem when open even when it has no children. I've removed this, hopefully nothing will break